go/oprf/groups/ecgroup/ec.go

```diff
@@ -14,13 +14,21 @@ import (
 	"github.com/alxdavids/oprf-poc/go/oprf/utils"
 	"github.com/alxdavids/oprf-poc/go/oprf/utils/constants"
 	"github.com/cloudflare/circl/ecc/p384"
+
+	"github.com/otrv4/ed448"
+	p448 "github.com/otrv4/ed448"
```

Comment:
This looks like a duplicated dependency.

Reply:
Solved ;)